Inria's gforge plattform has been returning 403 for a while, use debian
mirror for now to unblock builders.

diff -upr opam-2.0.8.orig/src_ext/Makefile.sources opam-2.0.8/src_ext/Makefile.sources
--- opam-2.0.8.orig/src_ext/Makefile.sources	2021-10-26 19:32:05.606498143 +0200
+++ opam-2.0.8/src_ext/Makefile.sources	2021-10-26 19:34:40.165349733 +0200
@@ -23,12 +23,12 @@ MD5_ocamlgraph = 9d71ca69271055bd22d0dfe
 
 $(call PKG_SAME,ocamlgraph)
 
-URL_cudf = https://gforge.inria.fr/frs/download.php/36602/cudf-0.9.tar.gz
+URL_cudf = https://deb.debian.org/debian/pool/main/c/cudf/cudf_0.9.orig.tar.gz
 MD5_cudf = a4c0e652e56e74c7b388a43f9258d119
 
 $(call PKG_SAME,cudf)
 
-URL_dose3 = https://gforge.inria.fr/frs/download.php/file/36063/dose3-5.0.1.tar.gz
+URL_dose3 = https://deb.debian.org/debian/pool/main/d/dose3/dose3_5.0.1.orig.tar.gz
 MD5_dose3 = e7d4b1840383c6732f29a47c08ba5650
 
 $(call PKG_SAME,dose3)
